src/Datadog.Trace.DuckTyping/Datadog.Trace.DuckTyping.csproj

```diff
@@ -19,7 +19,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="System.Reflection.Emit" Version="4.7.0" />
-    <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.7.0" />
+    <PackageReference Include="System.Reflection.Emit" Version="4.3.0" />
```

Comment:
@tonyredondo FYI I brought the NuGet package reference down since this still has the API surface area

Reply:
Oh yeah, makes sense.